lib/id3c/utils.py

```diff
@@ -90,6 +90,29 @@ def shorten(text, length, placeholder):
         return text
 
 
+def shorten_left(text, length, placeholder):
```

Comment:
Out of curiosity, what's the thought process behind making this a new function rather than adding a parameter to `shorten()`?

Reply:
Good question. I don't recall considering that when writing this (although the commit's author date was a month ago, so memory might be faulty). I think my default was to have a separate parallel function because the core of the function is substantially different, but I do see now that the supporting code is identical. We could adapt it if you prefer!

Reply:
Definitely fine to keep as is, I have no preference 🤷